tfe/resource_tfe_organization_test.go

```diff
@@ -0,0 +1,207 @@
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckTFEOrganizationExists(
+	n string, org *tfe.Organization) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		tfeClient := testAccProvider.Meta().(*tfe.Client)
+
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("No instance ID is set")
+		}
+
+		o, err := tfeClient.Organizations.Read(ctx, rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		if o.Name != rs.Primary.ID {
+			return fmt.Errorf("Organization not found")
+		}
+
+		*org = *o
+
+		return nil
+	}
+}
+
+func testAccCheckTFEOrganizationAttributes(
+	org *tfe.Organization) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if org.Name != "terraform-test" {
+			return fmt.Errorf("Bad name: %s", org.Name)
+		}
+
+		if org.Email != "admin@company.com" {
+			return fmt.Errorf("Bad email: %s", org.Email)
+		}
+
+		if org.SessionTimeout != 20160 {
+			return fmt.Errorf("Bad session timeout: %d", org.SessionTimeout)
+		}
+
+		if org.SessionRemember != 20160 {
+			return fmt.Errorf("Bad session remember: %d", org.SessionRemember)
+		}
+
+		if org.CollaboratorAuthPolicy != tfe.AuthPolicyPassword {
+			return fmt.Errorf("Bad auth policy: %s", org.CollaboratorAuthPolicy)
+		}
+
+		return nil
+	}
+}
+
+func testAccCheckTFEOrganizationAttributesUpdated(
+	org *tfe.Organization) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if org.Name != "terraform-updated" {
+			return fmt.Errorf("Bad name: %s", org.Name)
+		}
+
+		if org.Email != "admin-updated@company.com" {
+			return fmt.Errorf("Bad email: %s", org.Email)
+		}
+
+		if org.SessionTimeout != 3600 {
+			return fmt.Errorf("Bad session timeout: %d", org.SessionTimeout)
+		}
+
+		if org.SessionRemember != 3600 {
+			return fmt.Errorf("Bad session remember: %d", org.SessionRemember)
+		}
+
+		if org.CollaboratorAuthPolicy != tfe.AuthPolicyPassword {
+			return fmt.Errorf("Bad auth policy: %s", org.CollaboratorAuthPolicy)
+		}
+
+		return nil
+	}
+}
+
+func testAccCheckTFEOrganizationDestroy(s *terraform.State) error {
+	tfeClient := testAccProvider.Meta().(*tfe.Client)
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "tfe_organization" {
+			continue
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("No instance ID is set")
+		}
+
+		_, err := tfeClient.Organizations.Read(ctx, rs.Primary.ID)
+		if err == nil {
+			return fmt.Errorf("Organization %s still exists", rs.Primary.ID)
+		}
+	}
+
+	return nil
+}
+
+const testAccTFEOrganization_basic = `
+resource "tfe_organization" "foobar" {
+  name = "terraform-test"
+	email = "admin@company.com"
+}`
+
+const testAccTFEOrganization_update = `
+resource "tfe_organization" "foobar" {
+  name = "terraform-updated"
+	email = "admin-updated@company.com"
+	session_timeout = 3600
+	session_remember = 3600
+}`
```

Comment:
_minor_ can we make the spacing consistent?

Reply:
Whoeps! Yes, of course 👍 

Fixed in the next commit.